### PR TITLE
Added option to write audio book in single file

### PR DESCRIPTION
**Pull request by tpaktop77**
Also did some PEP8 refactoring

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -111,11 +129,22 @@ def downloader_main(output_dir, do_overwrite, audio_book_url):
 
     for url, fname in playlist:
         click.echo('Downloading chapter "{}"'.format(fname))
-        with open('{}.mp3'.format(os.path.join(full_path_dir, fname)), 'wb') as outfile:
+        if one_file:
+            file_name_and_path = '{}.mp3'.format(
+                get_audiobook_name(audio_book_url)
```
Comment (deathtracktor) [inline app.py]: We lost the `full_path_dir` here (mind combination of path and `--onefile` option).

File: app.py [hunk 2]
```diff
@@ -72,10 +80,14 @@ def get_full_dirname(dirname, do_overwrite):
...
+            click.echo('\n{} exists, and is not a directory!\n'.format(
+                full_path_dir)
+            )
             exit(1)
         if os.listdir(full_path_dir) and not do_overwrite:
-            if not click.confirm('\nDirectory "{}" exists. Overwrite?'.format(full_path_dir)):
+            if not click.confirm('\nDirectory "{}" exists. Overwrite?'.format(
+                    full_path_dir)
+            ):
                 sys.exit(1)
```
Comment (deathtracktor) [inline app.py]: One longer line is much  better here.

**Review by deathtracktor — CHANGES_REQUESTED** [review]

File: app.py (hunk 2 above)
Comment (tpaktop77) [reply app.py]: Is it okay that it is violating PEP8? 
If yes, I can revert this.

File: app.py (hunk 1 above)
Comment (tpaktop77) [reply app.py]: Okay, I was thinking to write it without additional directory, but this is also an option. I will fix it.

File: app.py (hunk 2 above)
Comment (deathtracktor) [reply app.py]: We can only violate a rule that is enforced :) The purpose of PEP8 is to improve readability, not to cause unnecessary eye strain. [Here](https://www.youtube.com/watch?v=wf-BqAjZb8M) is an excellent talk given by [Raymond Hettinger](https://twitter.com/raymondh?lang=en) on this topic.

Comment (tpaktop77) [reply app.py]: I've got your point and I some kind of agree, except that in most project I was or working it won't be allowed by CI/CD to pass checks. :)

**Pull request merged**